modules/ROOT/pages/contributing/source-git.adoc

```diff
@@ -4,55 +4,69 @@ include::partial$attributes.adoc[]
 :toc:
 
 If you’re familiar with how packaging is done in CentOS and Fedora, you may know dist-git repositories: each repository represents a source RPM package and contains all the RPM source files (RPM spec file, upstream tarball checksum and optionally patch files and other files).
-But the repository itself doesn't contain source code - you need to execute `%prep` phase to retrieve it from the cache.
+But the repository itself doesn't contain any source code - you need to execute `%prep` phase to see the source tree before it's being compiled.
 
 CentOS Stream 8 and CentOS Stream 9 have orthogonal relationship with Red Hat Enterprise Linux and because of this fact, source-git repositories work differently for both `c8s` and `c9s`.
 
-With CentOS Stream 8 source-git, the sources are unpacked and downstream code changes are layered on top as additional commits. The repositories don't contain upstream git history, instead we are unpacking an archive for the corresponding upstream release and this is the first commit. These are tracked as `c8s`, are updated continuously from https://git.centos.org/rpms[git.centos.org]. This means that merge requests cannot be merged here and contirubtions need to go through bugzilla and processed internally first.
+For more details on how to work in a source-git repository, please head on to https://packit.dev/docs/source-git/work-with-source-git/[packit's documentation].
```

Comment:
I think this sentence would have a better place at the end of the Stream 9 chapter.

Reply:
yup, definitely better a fit to move it